seismostats/analysis/estimate_mc.py

```diff
@@ -44,6 +44,8 @@ def cdf_discrete_GR(
 
 def empirical_cdf(
     sample: np.ndarray | pd.Series,
+    mc: float = None,
+    delta_m: float = 0,
```

Comment:
0 does not make sense, put a warning that binning is in fact needed
Default small (e-16? +warning that can take long)